woocommerce/woocommerce-paypal-payments
# set 3DS contingencies to 'SCA_WHEN_REQUIRED'

### Description

Do not enforce the 3D Secure challenge always. Use it only when required.
### Steps to test:


1. Add product to the cart.
2. Go to checkout and pay with credit card.
Expected: successful payment and order creation.

### Documentation




### Changelog entry
> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

* Fix - Do not enforce 3D Secure challenge when not necessary.
